src/lev-report/charts/HourlyUsage.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { VictoryAxis, VictoryChart, VictoryLabel, VictoryLine, VictoryTheme } from 'victory';
+
+const hours = Array.from({ length: 25 }, (_, h) => h);
+
+const HourlyUsage = ({ traces }) => traces && traces.length &&
+<VictoryChart
+	padding={{ top: 10, bottom: 50, left: 50, right: 5 }}
+	domainPadding={20}
+	theme={VictoryTheme.material}
+	style={{ labels: { fontSize: 10 } }}
+	height={250}
+	width={300}
+>
+	<VictoryAxis
+		domain={[0, 24]}
+		tickValues={[0, ...hours.filter(n => !(n % 3))]}
```

Comment:
Should the `.filter` line be moved out of the function?

Reply:
Yes, probably not much point in recalculating something that's never going to change!

Reply:
Fixed

Reply:
Did you remember to push? It doesn't seem to have updated.